cypress/integration/AlertStack/Shows_alertbars.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Shows AlertBars
```

Comment:
One point that is important (see design specs) is that the AlertBars are displayed in reversed order (i.e. this will mean the most recent one is at the top). Perhaps we could just check that the div has these CSS rules?
```css
display: flex;
flex-direction: column-reverse;
```

Reply:
I think we could probably also assert the order of the bars via their contents, seems more reliable. I'll see what I can do.

Reply:
As discussed this will also be done via a visual test.